### PR TITLE
Finish CeEntity implementations

### PR DESCRIPTION
**Pull request by lkno0705**

### CODE REVIEW

File: src/calculationEngine/entitys/Beasts.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
+package calculationEngine.entitys;
```
Comment (BigKahuna7385) [inline src/calculationEngine/entitys/Beasts.java:1]: Typo in the word entities

Comment (lkno0705) [reply src/calculationEngine/entitys/Beasts.java]: fixed

File: src/calculationEngine/entitys/CeEntity.java [hunk 2]
```diff
@@ -70,15 +95,16 @@ private int rollSalt() {
             }
         }
         for (int i = 0; i < pickedAttacks.length; i++) {
-            pickedAttacks[i] = new Attack(typedAttacks.get(random.nextInt(typedAttacks.size())));
+            int index = random.nextInt(typedAttacks.size());
+            pickedAttacks[i] = new Attack(typedAttacks.get(index));
```
Comment (BigKahuna7385) [inline src/calculationEngine/entitys/CeEntity.java:99]: pickedAttacks[i] = new Attack(typedAttacks.remove(index));

File: src/calculationEngine/entitys/CeEntity.java [hunk 3]
```diff
@@ -70,15 +95,16 @@ private int rollSalt() {
...
+            typedAttacks.remove(index);
```
Comment (BigKahuna7385) [inline src/calculationEngine/entitys/CeEntity.java:100]: You don't need this.

Comment (lkno0705) [reply src/calculationEngine/entitys/CeEntity.java:100]: You 100 % Sure?

File: src/calculationEngine/entitys/CeEntity.java [hunk 4]
```diff
@@ -15,6 +17,7 @@
     private BeastTypes type;
     private Nature nature;
     private CeEntity development;
+    private int developmentLvl;
```
Comment (BigKahuna7385) [inline src/calculationEngine/entitys/CeEntity.java:20]: Why is it called development and develomentLvl?

Comment (lkno0705) [reply src/calculationEngine/entitys/CeEntity.java:20]: Because I'm flash at typing

**Review by BigKahuna7385 — CHANGES_REQUESTED** [review]
Some minor issues, look at comments for details.

File: src/calculationEngine/entitys/CeEntity.java (hunk 3 above)
Comment (BigKahuna7385) [reply src/calculationEngine/entitys/CeEntity.java:100]: https://www.tutorialspoint.com/java/util/arraylist_remove.htm

**Review by BigKahuna7385 — APPROVED** [review]

**Pull request merged**